monai/visualize/class_activation_maps.py

```diff
@@ -204,12 +205,9 @@ def compute_map(self, x, class_idx=None, layer_idx=-1):
 
     def _upsample_and_post_process(self, acti_map, x):
         # upsampling and postprocessing
-        if self.upsampler:
-            img_spatial = x.shape[2:]
-            acti_map = self.upsampler(img_spatial)(acti_map)
-        if self.postprocessing:
-            acti_map = self.postprocessing(acti_map)
-        return acti_map
+        img_spatial = x.shape[2:]
```

Comment:
this change will require the previous user's input `upsampler=None` being changed to something like `upsampler=torch.nn.Identity`, but I think it should be a minor change.